src/core/tracking/gc.rs

```diff
@@ -26,6 +26,10 @@ pub(crate) struct TrackingGC {
     limit: u64,
     // The name of the thread
     name: String,
+    // Internal configuration map (optional). Here to keep a reference, not to
+    // be used.
+    #[allow(dead_code)]
+    config_map: Option<libbpf_rs::MapHandle>,
```

Comment:
I don't think using the thread to hold on to arbitrary useless maps is very clean.

No module should need to use this as they are stateful objects that can store data, maybe we should do the same with this "core" modules or think of a better way for them to keep data around.